components/03-sections/07-quickfacts/quickfacts--5up.hbs

```diff
@@ -0,0 +1,18 @@
+<section class="quick-fact-wrapper">
+    <div class="container">
+        <div class="row">
+            <div class="offset-lg-1 col-lg-11">
+                <div class="row">
+            {{#each items}}
+            <div class="col-xxl-2 col-xl-2 col-lg-2 col-md-2 col-sm-12">
```

Comment:
I was hoping the first item could have an offset. You could do this by using an if statement to add an offset to the first one: 
`
                {{#if @first}}
                    First item
                {{else}}
                    Not first item
                {{/if}}
`

Reply:
After reviewing this further with the offset, the content is not centered so I changed it to remove the offset and make the col-lg-12 row centered.
<img width="1836" alt="Screenshot 2023-11-29 at 12 55 17 PM" src="https://github.com/utsa-asc/college-dls/assets/92815346/36cae421-10a3-41dd-9f9b-8e35846212bc">